cogs/search.py

```diff
@@ -0,0 +1,301 @@
+                return
+
+            # Gets the first entry's stuff
+            first_title = self.tomd.handle(results[0]["title"]).rstrip("\n")
+            first_url = results[0]["url"]
+            first_desc = self.tomd.handle(results[0]["desc"]).rstrip("\n")
+
+            # Builds the substring for each of the other results.
+            other_results: List[str] = []
+            for r in results[1:count]:
+                title = self.tomd.handle(r["title"]).rstrip("\n")
+                url = r["url"]
+                other_results.append(f"**{title}** {url}")
+            other_msg: str = "\n".join(other_results)
+
+            # Builds message
+            msg = (
+                f"Showing **{count}** results for `{query_display}`.\n\n"
+                f"**{first_title}** {first_url}\n{first_desc}\n\n"
+                f"{other_msg}\n\n_Powered by Qwant._"
+            )
+
+            msg = re.sub(r"(https?://(?:www\.)?[-a-zA-Z0-9@:%._+~#=]+\." r"[a-zA-Z0-9()]+\b[-a-zA-Z0-9()@:%_+.~#?&/=]*)", r"<\1>", msg)
+
+            # Sends message
+            self.info(f"**New Search** - `{ctx.author}` in `{ctx.guild}`\n\n{msg}", name="New Search")
+            await ctx.send(msg)
+
+    @command()
+    async def search(self, ctx: Context, *, query: str) -> None:
+        """Search online for general results."""
+
+        await self._basic_search(ctx, query)
+
+    @command(aliases=["video"])
+    async def videos(self, ctx: Context, *, query: str) -> None:
+        """Search online for videos."""
+
+        await self._basic_search(ctx, query, "videos")
+
+    @command()
+    async def music(self, ctx: Context, *, query: str) -> None:
+        """Search online for music."""
+
+        await self._basic_search(ctx, query, "music")
+
+    @command(aliases=["file"])
+    async def files(self, ctx: Context, *, query: str) -> None:
+        """Search online for files."""
+
+        await self._basic_search(ctx, query, "files")
+
+    @command(aliases=["image"])
+    async def images(self, ctx: Context, *, query: str) -> None:
+        """Search online for images."""
+
+        await self._basic_search(ctx, query, "images")
+
+    @command()
+    async def it(self, ctx: Context, *, query: str) -> None:
+        """Search online for IT-related information."""
+
+        await self._basic_search(ctx, query, "it")
+
+    @command(aliases=["map"])
+    async def maps(self, ctx: Context, *, query: str) -> None:
+        """Search online for map information."""
+
+        await self._basic_search(ctx, query, "maps")
+
+    @Cog.listener()
+    async def on_command_error(self, ctx: Context, error: Error) -> None:
+        """Listener makes no command fallback to searching."""
+        fallback = (CommandNotFound, CheckFailure)
+
+        if isinstance(error, fallback):
+            try:
+                await self._basic_search(ctx, ctx.message.content[len(ctx.prefix) :])  # conflict bw flake8 nd black here
```

Comment:
```suggestion
                await self._basic_search(ctx, ctx.message.content[len(ctx.prefix):])
```
Prefer `flake8` over `black`